plugins/repositories/schema/system-description-repositories.schema-v10.json

```diff
@@ -37,7 +37,6 @@
               },
               "url": {
                 "type": "string",
-                "format": "uri",
```

Comment:
I think Cornelius wanted to be less strict if there are issues. For most parts we only require strings with at least one character. We though url should work here but like I mentioned Debian does not follow this schema.

I am not sure if there is a schema for the Debian cdrom urls. If not I am not sure how `oneOf` would help.